### DISCUSSION TITLE
What 🌱 does saffron come from?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#905] [Galaxy Trivia] [answered]
**Bonus:** 1lb of it is worth more than 1lb of 🥇. Prove mathematically whether it's true or false.

**Comment by peterbaccaro** [comment, marked as answer]
Saffron comes from the **saffron crocus plant, scientifically known as Crocus sativus**. It is a flowering plant that belongs to the iris family (Iridaceae). Saffron is derived from the stigma and styles of the flower, which are collected and dried to produce the highly prized spice known as saffron. Each saffron crocus flower produces only a few threads of saffron, making it one of the most expensive spices in the world.

The US gold price per gram today is $62.93.
The **price per lb for US gold** is therefore 62.93 * 453.592 = **$28,499.47**

The saffron price per gram in the USA is $1490. 
You need almost 150 flowers of saffron to make just 1 gram of it. and there are different kind of saffron that is categorized in grade. 
The **price in lb for saffron in the USA** is therefore 1490 * 453.592 = **$675,910.28**

Therefore 1lb of saffron is worth more than 1lb of gold.

**Reply by ParanoidUser** [reply]
> Gold price per gram today is USD 62.93 which means the price per pound(lb) is USD 0.1386.

How to get rich quickly?
- Step 1: Buy 35 pounds of gold for $5 USD.
- Step 2: Sell at $62.93 USD per gram.
- Step 3: Split the profit. 🤑

*This is not financial advice.*

**Reply by peterbaccaro** [reply]
Maybe my conversion was slightly out. :laughing:
I have updated my answer.

**Reply by ParanoidUser** [reply]
> The price in lb for saffron in the USA is therefore 1490 * 453.592 = $675,910.28

How to get poor quickly?

- Step 1: Buy 1 gram of saffron for $1490 USD.
- Step 2: Go to any grocery store and find it on [BOGO for $10](https://www.goldensaffron.com).
- Step 3: Take a hard pill to swallow. 😢

*This is not financial advice.*

**Reply by peterbaccaro** [reply]
The price of saffron depends on the quality. I used this source for the price.

www.eworldtrade.com/blog/saffron-price-per-gram-in-usa/amp/

If the price of saffron is $10 per gram then the price per pound (lb) is 10 * 453.592 = $4,535.92.

The price per lb for US gold is 62.93 * 453.592 = $28,499.47.

The answer in this case is **False** as gold would be worth more than saffron.

**Comment by raeef1001** [comment]
Saffron comes from the saffron crocus plant, scientifically known as Crocus sativus. It is a flowering plant that belongs to the iris family (Iridaceae). Saffron is derived from the stigma and styles of the flower, which are collected and dried to produce the highly prized spice known as saffron. Each saffron crocus flower produces only a few threads of saffron, making it one of the most expensive spices in the world.

The US gold price per gram today is $62.93.
The price per lb for US gold is therefore 62.93 * 453.592 = $28,499.47

The saffron price per gram in the USA is $1490.
You need almost 150 flowers of saffron to make just 1 gram of it. and there are different kind of saffron that is categorized in grade.
The price in lb for saffron in the USA is therefore 1490 * 453.592 = $675,910.28

Therefore 1lb of saffron is worth more than 1lb of gold.
